### PR TITLE
Fixes #233, removed built-in extensions

### LINKED ISSUE #233: Remove built-in extensions
(From #153) These extensions are far out of date, and we haven't come up with any general solution to automatically updating them (or the lists they rely on). Instead, we should pull them out of OpenWPM entirely to lower the core platform maintenance cost.

Studies which need to use extensions can add them manually. This ensures that users of the platform grab up-to-date versions.

We should file a follow-up issue to explore list update / profile priming options as discussed in #153. While we likely won't want to bundle extensions, it would still be nice to provide an interface to add a new extension and prime a profile with any lists it requires.

### PR DESCRIPTION
**Pull request by Metropass**
Removed from `configure_firefox.py`:
`ghostery`
`uBlock Origin`
`https-everywhere`

I also removed the whitespace to save a few bytes. I checked `deploy_browser.py`, `deploy_firefox.py`, and `selenium_firefox.py` just to make sure the removed extensions aren't referenced, but I found nothing.

I deleted the references within `default_browser_params.json`

### CODE REVIEW

File: automation/default_browser_params.json [hunk 1]
```diff
@@ -17,13 +16,8 @@
     "display_mode": "native",
     "browser": "firefox",
     "prefs": {},
-
     "tp_cookies": "always",
     "donottrack": false,
-    "disconnect": false,
-    "ghostery": false,
-    "https-everywhere": false,
     "adblock-plus": false,
```
Comment (vringar) [inline automation/default_browser_params.json:21]: Just realizing, that we also have adblock-plus in here. Feel free to delete that as well.

**Comment by vringar** [comment]
Could you please also delete this entire folder https://github.com/mozilla/OpenWPM/tree/master/automation/DeployBrowsers/firefox_extensions

**Comment by vringar** [comment]
Also I'm disabling the localstack tests for now, as I can't find a fix. This means you'll have to merge the master branch into your branch once #756 gets merged.

**Comment by Metropass** [comment]
> 
> 
> Also I'm disabling the localstack tests for now, as I can't find a fix. This means you'll have to merge the master branch into your branch once #756 gets merged.

Noted, will do!

**Review by vringar — APPROVED** [review]

**Comment by vringar** [comment]
LGTM. Do you have anything you still want to do?

**Comment by Metropass** [comment]
> 
> 
> LGTM. Do you have anything you still want to do?

In terms of this PR, I think we got everything, but I was wondering if I can assist in issue #743 ? @vringar

**Review by vringar — CHANGES_REQUESTED** [review]
I just realize we also don't need
https://github.com/mozilla/OpenWPM/blob/2b1a54eff38feebb737abf402382156e7b2a7438/automation/DeployBrowsers/configure_firefox.py#L19-L22
any more.  Which means we should also remove
https://github.com/mozilla/OpenWPM/blob/2b1a54eff38feebb737abf402382156e7b2a7438/automation/DeployBrowsers/configure_firefox.py#L4-L7

**Comment by vringar** [comment]
Regarding #743 I'd wait a bit to see if the person currently assigned on it responds, but feel free to have a look at it.
If you want to spend more time on OpenWPM in general, I can also go through the issues and ping you on stuff that's interesting.

**Comment by Metropass** [comment]
> 
> 
> Regarding #743 I'd wait a bit to see if the person currently assigned on it responds, but feel free to have a look at it.
> If you want to spend more time on OpenWPM in general, I can also go through the issues and ping you on stuff that's interesting.

Yeah that'll be great, thank you so much! 😄

**Comment by Metropass** [comment]
> 
> 
> I just realize we also don't need
> 
> https://github.com/mozilla/OpenWPM/blob/2b1a54eff38feebb737abf402382156e7b2a7438/automation/DeployBrowsers/configure_firefox.py#L19-L22
> 
> any more. Which means we should also remove
> 
> https://github.com/mozilla/OpenWPM/blob/2b1a54eff38feebb737abf402382156e7b2a7438/automation/DeployBrowsers/configure_firefox.py#L4-L7

Noted, I'll take care of that.

**Review by vringar — APPROVED** [review]

**Pull request merged**